src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/Schemas/JSON/template.json

```diff
@@ -440,10 +440,7 @@
               },
               "args": {
                 "description": "A list of key-value pairs to use when performing the action. The specific parameters required / allowed are defined by the action itself.",
-                "type": "object",
```

Comment:
should we add string array instead of removing `additionalProperties`?

Reply:
Honestly I'm not sure what `additionalProperties` is supposed to do, but it conflicted with `array` type, and was showing as error...

Reply:
As discussed, probably related to `IPostAction` interface which only support strings for the args